nomad/nomad_jobs/model_bazaar_job.hcl.j2

```diff
@@ -45,14 +45,9 @@ job "modelbazaar" {
       }
 
       config {
-        image = "thirdaistaging.azurecr.io/model-bazaar:latest"
+        image = "{{ PRIVATE_SERVER_IP }}:5000/model-bazaar"
```

Comment:
Just like these image names are being changed, we also need to change the image names for train, rlhf update, and deploy hcl files in the neuraldb-enterprise service repo

Reply:
yeah yeah